docs/development/troubleshooting.mdx

```diff
@@ -63,3 +60,16 @@ for head in headers:
 ```
 This is especially helpful in databases like Snowflake, which don't have an `iferror()` function to check for missing columns.
 
+
+---
+
+## `InvalidToken` error in a Mage deployment on AWS
+
+When deploying Mage to AWS, you may see an error like this:
```

Comment:
This is not for deploying Mage to AWS.
This is an error of accessing data from S3.

Reply:
Oh, good catch. Thank you